docs/inputs/table-group.md

```diff
@@ -17,26 +17,106 @@ Similar to an [**Input Group**](input-group), a **Table Group** allows the _**de
+For example, the following **Table Group** has three columns and 1 row. The header can be made using RichText input type and does NOT count as a row. The array name available for the **Code** step is `table`.
 
 <div style={{textAlign: 'center'}}>
 
-![img alt](/docs/inputs/table_group.png)
+![img alt](/docs/inputs/table_group_numbered.png)
 
 </div>
 
+1. Drag the **Table Group** to order it vertically amongst the other groups
+2. Edit the **Table Group** properties (described below)
+3. Delete the **Table Group**
+4. Duplicate the last row in the current **Table Group** (similar to duplicate input but for the whole row)
+
+### Editing a table group
+
+To edit the table group, click the :pencil2: icon near the top right side of the table group. A popup will appear with the following options:
+
+<div style={{textAlign: 'center'}}>
+
+![img alt](/docs/inputs/edit_table_group_dialog.png)
+
+</div>
+
+1. **Title** -> the title of the table group
+2. **Name** -> this is the variable name of the table which will be used in the python code
+3. **Columns** -> the number of columns in the table
+
+| :trophy: The following are [premium](https://mecsimcalc.com/pricing) features |
+| ----------------------------------------------------------------------------- |
+
+4. **Dynamic Rows** -> check this box if you want the user to be able to add and delete rows from the table  
+    After checking this box a list of options will appear:  
+   a) **_Max Rows_** -> The maximum number of rows that can be in the table (**NOT including the header row**)  
+   b) **_Min Rows_** -> The minimum number of rows that can be in the table
+   (**NOT including the header row**)  
+   c) **_Row Group Size_** -> The number of rows that are 'combined' together when the user 'Adds' or 'Deletes' rows. (_This should almost always be set to 1 ...unless your row is comprised of multiple lines_)
+5. **Conditional Input Group** -> [Learn more](/inputs/input-group#conditional-input-group)
+
 :::caution
-Don't forget to click on **APPLY** after changing the number of _columns_ or the array _name_ of the **Table Group**
+Don't forget to click on **APPLY** after editing the properties of the **Table Group**
+:::
+
+### Dynamic Rows
+
+The following embedded app demonstrates the **Dynamic Rows** feature using the **Table Group** properties shown above. Enter the values and press submit to calculate the weighted GPA.  
+Use the [dynamic table template](https://mecsimcalc.com/create) (under **The Basics**) to start creating a similar app!
+
+<div style={{width: "100%", height: "900px", overflow: "hidden"}}><iframe src="https://mecsimcalc.com/app/7646727/weighted_gpa_calculator_mobile_friendly" width="100%" height="100%" title="MecSimCalc" style={{position:"relative", left:"-45px", top:"-48px"}} frameBorder="0"></iframe></div>
```

Comment:
For this app, can you pre-fill the initial values so i can click submit without getting this zero division error?

<img width="520" alt="image" src="https://github.com/MecSimCalc/MecSimCalc-docs/assets/57570491/b2635a7c-468d-4f64-a915-53f3b9b42b68">